BlackJackGame/Game.cs

```diff
@@ -138,19 +148,114 @@ public int CalculateScore(List<string> playersHand)
         }
         
 
-        public List<string> DealersTurn(List<string> dealersHand, IDeck deck)
+        public int DealersTurn(List<Card> dealersHand, IDeck deck, int score)
```

Comment:
what is the int score parameter that is being passed into DealersTurn?

Reply:
It is the initial score for the dealers hands, so the total score after first two cards are drawn.